newa/cli.py

```diff
@@ -416,7 +416,7 @@ def apply_release_mapping(string: str,
     if not mapping:
         mapping = [
             r'\.GA$=',
-            r'\.Z\.?(MAIN)?(\+)?(AUS|EUS|E4S|TUS)?$=',
+            r'\.Z\.?(MAIN)?(\+)?(AUS|E.?S|TUS)?$=',
```

Comment:
I am sorry, I have just realized. `E.?S` matches also `ES`. Do we actually want `E.S`?